host_core/mix.exs

```diff
@@ -16,7 +16,7 @@ defmodule HostCore.MixProject do
   def application do
     [
       extra_applications: [:logger, :crypto],
-      mod: {HostCore.Application, []}
+      mod: {HostCore, []}
```

Comment:
As per the other comment, I'm not comfortable with removing the `Application` module because the use of this module is not only the default for scaffolded applications but it is the de facto standard in idiomatic Elixir applications.